config/rewrites.d/support.rackspace.com.json

```diff
@@ -916,6 +916,13 @@
         "to": "/how-to/article-retired/",
         "rewrite": false,
         "status": 301
+      },
+      {
+        "description": "Retire General Info in the Cloud Control Panel",
+        "from": "^\\/how-to\\/general-information-cloud-office-control-panel(.*)$",
+        "to": "/how-to/cloud-office-control-panel-overview/",
```

Comment:
Should be: "to": "/how-to/article-retired/",

Reply:
@catlook Please see Nate's comment on the retire PR at https://github.com/rackerlabs/rackspace-how-to/pull/2507#issuecomment-369707436. He said to redirect to this content, which is similar.